assets/src/components/AchievedGoals.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <div class="index">
+    <h1>Achieved Goals</h1>
+    <div class="list-of-achieved-goals">
+      <ul id="achieved-goals">
+        <li v-for="goal in goals" :key="goal.id" :id="'goal-' + goal.id">
+          <i class="fas fa-check-circle"></i> {{ goal.activity }} in {{ goal.location }}.
+        </li>
+      </ul>
+    </div>
+  </div>
+</template>
+
+<script>
+import axios from 'axios'
+
+export default {
+  name: 'AchievedGoals',
+  props: {
+    msg: String
+  },
+  data () {
+    return {
+      goals: [],
+      error: null,
+      uri: process.env.VUE_APP_API_URI
+    }
+  },
+  created () {
+    this.fetchAchievedGoals()
+  },
+  methods: {
+    fetchAchievedGoals: async function () {
+      let achievedIndexUrl = this.uri + "/achieved"
+      this.goals = []
```

Comment:
Setting goals to `[]` is probably not necessary.

Reply:
Removed in eb6f13a